### ISSUE TITLE
nvidia-settings no longer required by any nvidia packgage

### ISSUE THREAD

**Issue by sndirsch** [openSUSE/nvidia-driver-G06#53]
Just noticed that nvidia-settings package is no longer required by any othe nvidia subpackgage. Not sure if this was by intention. It happened with PR #28.

**Comment by scaronni** [comment]
Yes it is intentional. Any information you can get with `nvidia-settings -q` can be derived from other tools.

In the specific case of a normal laptop (i.e. an Optimus laptop) with just `nvidia-video-G06` and nothing else it creates more confusion than benefits, as user thinks they need to use that to configure X.

With Wayland, has almost no use anymore, except for querying some information. I would live it as optional.

**Comment by sndirsch** [comment]
Thanks. That's great news. Nothing to change here! :-)
